nipoppy_cli/pyproject.toml

```diff
@@ -37,16 +37,16 @@ version = "1.0.0" # TODO eventually use dynamic versioning
 [project.optional-dependencies]
 dev = ["nipoppy[doc]", "nipoppy[test]", "pre-commit"]
 doc = [
-    "furo==2024.1.29",
-    "pygments-csv-lexer==0.1.3",
-    "sphinx==7.2.6",
-    "sphinx-argparse==0.4.0",
-    "sphinx-autoapi==3.0.0",
-    "sphinx-copybutton==0.5.2",
-    "sphinx-jsonschema==1.19.1",
-    "sphinx-togglebutton==0.3.2",
-    "mdit-py-plugins==0.4.0",
-    "myst-parser==2.0.0",
+    "furo>=2024.1.29",
+    "pygments-csv-lexer>=0.1.3",
+    "sphinx>=7.2.6",
+    "sphinx-argparse>=0.4.0",
+    "sphinx-autoapi>=3.0.0",
+    "sphinx-copybutton>=0.5.2",
+    "sphinx-jsonschema>=1.19.1",
+    "sphinx-togglebutton>=0.3.2",
+    "mdit-py-plugins>=0.4.0",
+    "myst-parser>=2.0.0",
```

Comment:
not sure if there was a reason to pin those to those versions but I think that >= should be enough, no?